source/Nuke.Common/Tooling/ToolPathResolver.cs

```diff
@@ -136,7 +137,7 @@ private static string GetPackageDirectory(string[] packageIds, [CanBeNull] strin
             }
             catch (Exception exception)
             {
-                if (!NuGetPackagesConfigFile.EndsWithOrdinalIgnoreCase(".csproj"))
+                if (NuGetPackagesConfigFile != null && !NuGetPackagesConfigFile.EndsWithOrdinalIgnoreCase(".csproj"))
```

Comment:
Preexisting bug - if `NuGetPackagesConfigFile` was null, it was masking the real exception.

Reply:
fixed in 6.0.2